data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -562,7 +563,7 @@ Potion (Oil of Magic Vestment +5)									OUTPUTNAME:[NAME]			TYPE:Magic.Potion.
 ###Block: Rings
 # Equipment Name							Unique Key								Output Name									Type			Cost		Weight	Modifier					Source Page		Ability										Innate Spells									Combat bonus				Bonus to skill					Multiply spell cast/day				Modify VAR								Special Property																																																																																																																																																																																																																																																																																																																																																																																																																																	SORTKEY
 Ring of Animal Friendship																								TYPE:Magic.Ring	COST:10800	WT:0								SOURCEPAGE:p.230												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=3|Charm Animal																															SPROP:On command, this ring affects an animal as if the wearer had cast charm animal. Faint enchantment; CL 3rd
-Ring of Blinking																										TYPE:Magic.Ring	COST:27000	WT:0								SOURCEPAGE:p.230												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=7|Blink																																SPROP:On command, this ring makes the wearer blink, as with the blink spell. Moderate transmutation; CL 7th
+Ring of Blinking																										TYPE:Magic.Ring	COST:27000	WT:0								SOURCEPAGE:p.230												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=7|Blink																																SPROP:On command, this ring makes the wearer blink, as with the blink spell (Physical attacks and targetted spells have 50% miss chance, your own spells have a 20% miss chance too. Half damage from area affects and falling. Can step through solid objects, 50% per 5' chance of becoming material and taking damage/being shunted to nearest open space , PHB P206). Moderate transmutation; CL 7th	 ABILITY:Special Ability|VIRTUAL|Concealment Chance	 BONUS:VAR|ConcealmentChance|50
```

Comment:
% needs an extra % otherwise the code parses this as a var substitution.

Reply:
on PDF output this does output % correctly, SPROP not working as documented unfortunately.